posts/2023-09-01-electing-new-project-directors.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Electing New Project Directors"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Today we are launching the process to elect new Project Directors to the Rust Foundation Board of Directors.
+As we begin the process, we wanted to spend some time explaining the goals and procedures we will follow.
+We will summarize everything here, but if you would like to you can read the [official process documentation][pde-process].
+
+We ask all project members to begin working with their Leadership Council representative to nominate potential Project Directors. See the [Candidate Gathering](#candidate-gathering) section for more details. Nominations are due by September 15, 2023.
+
+<!-- TODO: Once we've merged this PR we should link to the final location instead. -->
+[pde-process]: https://github.com/rust-lang/leadership-council/pull/12
+
+## What are Project Directors?
+
+The Rust Foundation Board of Directors has five seats reserved for Project Directors.
+These Project Directors serve as representatives of the Rust project itself on the Board.
+Like all Directors, the Project Directors are elected by the entity they represent, which in the case of the Rust Project means they are elected by the Rust Leadership Council.
+Project Directors serve for a term of two years and will have staggered terms.
+This year we will appoint two new directors and next year we will appoint three new directors.

```

Comment:
Is it ok if we put the names of the directors that are stepping down this year into this post? (and maybe remind people who the other three are too?) I think that'll be one of the questions people reading this immediately have, but if there's a reason we don't want to put that info in here, that's fine.

Reply:
I can't find it in Zulip, but I raised this at one point and @rylev (I think, I could be misremembering) suggested we should wait to say that when we announce the results of the election. But, all the comments I could find from people say the opposite, so maybe I'll include something.

Reply:
I added the names of the current project directors as well as the ones who will be rotating out.